scripts/generateProptypes.ts

```diff
@@ -117,8 +127,13 @@ async function run() {
 
       return fileName === folderName;
     });
+  // I believe that the program still needs every file
+  // for import-extends. Not sure but compiling all to be sure
```

Comment:
I think it only follows the imports in the specified files, so if you have overrides that aren't explicitly imported you need to list all of them.

Reply:
It does currently pick up props from imported files. Even the ones in node_modules. I think I just added it for debugging. I'll remove it and see if this changes anything.

Reply:
removed.